Graphs/FordFulkerson.js

```diff
@@ -0,0 +1,52 @@
+// https://en.wikipedia.org/wiki/Ford%E2%80%93Fulkerson_algorithm
+function fordFulkerson(capacity, source, sink) {
```

Comment:
It needs to be documented what format this is in. (Side note: I would prefer an adjacency list representation because it yields a better time complexity.)